metricbeat/module/elasticsearch/elasticsearch_integration_test.go

```diff
@@ -59,77 +57,72 @@ var metricSets = []string{
 	"shard",
 }
 
-func TestFetch(t *testing.T) {
-	compose.EnsureUp(t, "elasticsearch")
-
-	host := net.JoinHostPort(getEnvHost(), getEnvPort())
-	err := createIndex(host)
-	assert.NoError(t, err)
-
-	err = enableTrialLicense(host)
-	assert.NoError(t, err)
-
-	err = createMLJob(host)
-	assert.NoError(t, err)
-
-	for _, metricSet := range metricSets {
-		checkSkip(t, metricSet, host)
-		t.Run(metricSet, func(t *testing.T) {
-			f := mbtest.NewReportingMetricSetV2(t, getConfig(metricSet))
-			events, errs := mbtest.ReportingFetchV2(f)
-
-			assert.Empty(t, errs)
-			if !assert.NotEmpty(t, events) {
-				t.FailNow()
-			}
-			t.Logf("%s/%s event: %+v", f.Module().Name(), f.Name(),
-				events[0].BeatEvent("elasticsearch", metricSet).Fields.StringToPrint())
-		})
+func TestElasticsearch(t *testing.T) {
+	runner := compose.TestRunner{
+		Service: "elasticsearch",
+		Options: compose.RunnerOptions{
+			"ELASTICSEARCH_VERSION": {
```

Comment:
@ycombinator you will like that

Reply:
I easilly reproduced the failure on 6.5.1 with this :slightly_smiling_face: